dtfs-central-api/api-tests/v1/utilisation-reports/post-validate-utilisation-report-data.api-test.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpStatusCode } from 'axios';
+import { testApi } from '../../test-api';
+
+console.error = jest.fn();
+
+const URL = '/v1/utilisation-reports/report-data-validation';
+
+describe(`POST ${URL}`, () => {
+  it('returns a 200 when request body is valid', async () => {
```

Comment:
could we format this so it is `returns a ${HttpStatusCode.Ok} when request body is valid` and the same for the rest in this file please